### ISSUE TITLE
Apps with a splash window throws off layout

### ISSUE THREAD

**Issue by DavidMtbg** [TimUntersberger/wwm#60]
I had Chrome in a workspace and then launched Excel.  This resulted in three tiles instead of two, with the middle one being an empty tile where I guess the splash window would have been.

**Comment by TimUntersberger** [comment]
I have a very bad feeling that this might be another veryyyyy specific problem.

I looked at an image of the splashscreen and they might have implemented the splash screen as a regular window.

After starting the real window they may just be hiding the splashscreen and not closing it. There is currently a problem with the native powershell terminal where a similiar thing happens.

Have you tried adding a rule like the following one: 

```yaml
pattern: ^<SPLASHSCREENNAME>$
manage: false
```

This might be a temporary fix for this. It could also be possible to create a pattern for every office splashscreen.

**Comment by DavidMtbg** [comment]
How would I get the splashscreen name?  I tried Excel and it didn't manage anything including the proper app.  I tried Microsoft Excel and it managed both the splashscreen and the proper app.

**Comment by TimUntersberger** [comment]
Does the Splashscreen not show in the taskbar? If not I guess it would be possible by looking at the logs. There should be various events listed where the SHOW events should be the most useful for you. 

Example
```
[2020-06-27 19:24:01 DEBUG wwm::event_handler::winevent] Show(false): 'wwm_app_bar' | 396104
```

You could also look for a draw grid log and directly underneath are the currently shown windows.

Example
```
[2020-06-27 19:23:30 DEBUG wwm::tile_grid] Drawing grid
[2020-06-27 19:23:30 DEBUG wwm::tile_grid] Tile(id: 264990, title: 'Untitled - Notepad', row: None column: None)
```

**Comment by TimUntersberger** [comment]
Were you able to find the name of the splashscreen window? I might have an idea how to fix windows that only 'hide' and don't really 'close'

**Comment by TimUntersberger** [comment]
Fixed by 96abbea in Development branch. Could you please try the Development Build together with excel?
